reporter/gcppubsub/pubsub.go

```diff
@@ -0,0 +1,110 @@
+package gcppubsub
+
+import (
+	"cloud.google.com/go/pubsub"
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/openzipkin/zipkin-go/model"
+	"github.com/openzipkin/zipkin-go/reporter"
+	"log"
+	"os"
+)
+
+const defaultPubSubTopic = "defaultTopic"
+
+var resultMsg = make(chan reporterResult)
```

Comment:
Just out of curiosity, why this is not part of the struct that entirely uses it? Also, could we rename it into `resultMsgC` otherwise it is the easiest to understand what happens in line 104.